content/html/concepts/tags/terms/doctype/doctype.md

```diff
@@ -0,0 +1,31 @@
+---
+Title: '<doctype>' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
```

Comment:
Great job finding using the templates! Let's actually remove these comments so we are just left with the necessary text.